vagrant/provisioning/roles/start-arkcase/tasks/main.yml

```diff
@@ -4,19 +4,6 @@
     path: "{{ root_folder }}/common/symmetricKey.decrypted"
     state: absent
 
-- name: start config server
-  become: yes
-  systemd:
-    daemon_reload: true
-    name: "config-server"
-    state: started
```

Comment:
Are we starting it somewhere else? Why don't you just put a when condition on arkcase_version?

Reply:
It should not run prior arkcase version 3.3.1